packages/modeling/src/measurements/measureBoundingBox.js

```diff
@@ -123,9 +119,10 @@ export const measureBoundingBox = (...geometries) => {
   if (geometries.length === 0) throw new Error('wrong number of arguments')
 
   const results = geometries.map((geometry) => {
-    if (path2.isA(geometry)) return measureBoundingBoxOfPath2(geometry)
-    if (geom2.isA(geometry)) return measureBoundingBoxOfGeom2(geometry)
-    if (geom3.isA(geometry)) return measureBoundingBoxOfGeom3(geometry)
+    if (path2.isA(geometry)) return measureCached(geometry, measureBoundingBoxOfPath2)
```

Comment:
It is caching still, I just did some refactoring.

I split out the caching code into a function called `measureCached` which takes a geometry and a measurement function as arguments.

It was tedious to have 4 copies of the caching code in each geometry sub-function (path, geom2, geom3, slice...). So that's why I factored out the caching code like this.